godot-core/src/export.rs

```diff
@@ -0,0 +1,163 @@
+/*
```

Comment:
This would probably fit better into the existing [`godot::bind`](https://godot-rust.github.io/docs/gdext/master/godot/bind/index.html) module, rather than its own top-level one.

(I know the name "bind" is not great; it's coming from binding. "export" is however specifically the variable-export, and not functions/classes/etc. We can discuss this later maybe 🙂)